liferay-echo/liferay-portal
# LPS-90594 Import fragments in a new page

Motivation
=======
When importing fragments or fragment's sets, we will open a new page were we will show the results of the importation process.
[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-190594)

Steps to Verify:
----------------
1. Set feature.flag.LPS-174939=true in portal-ext.properties
1. Go to design fragments and test importing a fragment set and a fragment inside a fragment set.
1. Test with the zip files included in this PR. For the "fragment error if not overwrite" file, first it is necessary to import the fragment in a fragment set and after that, import it again but unchecking the "Overwrite Existing Entries" option. 
1. Disable the Feature Flag and check that everything works as before


[fragments success.zip](https://github.com/liferay-echo/liferay-portal/files/12162833/fragments.success.zip)
[fragments success and warnings.zip](https://github.com/liferay-echo/liferay-portal/files/12162836/fragments.success.and.warnings.zip)
[fragment warning.zip](https://github.com/liferay-echo/liferay-portal/files/12162840/fragment.warning.zip)
[fragment error if not overwrite.zip](https://github.com/liferay-echo/liferay-portal/files/12162843/fragment.error.if.not.overwrite.zip)
